include/tapkee/routines/diffusion_maps.hpp

```diff
@@ -64,9 +64,10 @@ DenseSymmetricMatrix compute_diffusion_matrix(RandomAccessIterator begin, Random
 	p = diffusion_matrix.colwise().sum();
 
 	// compute full matrix as we need to compute sum later
+	// TODO add weighting alpha, use linear algebra to compute D^-alpha K D^-alpha
 	for (IndexType i=0; i<n_vectors; i++)
 		for (IndexType j=0; j<n_vectors; j++)
-			diffusion_matrix(i,j) /= pow(p(i)*p(j),timesteps);
+			diffusion_matrix(i,j) /= p(i)*p(j);
```

Comment:
It would be, but timesteps is wrong here. Timesteps only please a role for the scaling with the eigenvalues in the diffusion embedding. What is relevant here is the parameter alpha.